test/contestant_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require './lib/game'
+
+
+class ContestantTest < MiniTest::Test
+
+  def test_it_exists
+    alexander = Contestant.new({first_name: 'Alexander',
+                                      last_name: 'Aigiades',
+                                      age: 28,
+                                      state_of_residence: 'CO',
+                                      spending_money: 10})
+    assert_instance_of Contestant, alexander
+
+  end
+
+  def test_it_has_attributes
+    alexander = Contestant.new({first_name: 'Alexander',
+                                      last_name: 'Aigiades',
+                                      age: 28,
+                                      state_of_residence: 'CO',
+                                      spending_money: 10})
+
+    assert_equal "Alexander Aigiades", alexander.full_name
+    assert_equal 28, alexander.age
+    assert_equal "CO", alexander.state_of_residence
+    assert_equal 'Aigiades', alexander.last_name
+    assert_equal  10, alexander.spending_money
+    assert_equal false, alexander.out_of_state?
+
+  end
+
+  def test_can_have_game_interests
+    alexander = Contestant.new({first_name: 'Alexander',
+                                      last_name: 'Aigiades',
+                                      age: 28,
+                                      state_of_residence: 'CO',
+                                      spending_money: 10})
+    assert_equal [], alexander.game_interests
+    alexander.add_game_interest('Mega Millions')
+    alexander.add_game_interest('Pick 4')
+    assert_equal ["Mega Millions", "Pick 4"], alexander.game_interests
+
+
+  end
+
+  def test_charge_contestant
+    alexander = Contestant.new({first_name: 'Alexander',
+                                      last_name: 'Aigiades',
+                                      age: 28,
+                                      state_of_residence: 'CO',
+                                      spending_money: 10})
+  end
```

Comment:
i like the idea of putting this responsibility on the contestant class-- technically the interaction pattern asks for this to live on the lottery class, but i think you could have swung this with a bit more time.